package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/seo",
-  "version": "1.36.0",
+  "version": "1.34.2",
```

Comment:
Can we move back to master n update this again?

Reply:
Done.